tests/puppeteer/campaigns/functional/BO/customers/customers/01_filterAndQuickEditCustomers.js

```diff
@@ -186,7 +186,7 @@ describe('Filter And Quick Edit Customers', async () => {
       await this.pageObjects.customersPage.filterCustomers(
         'select',
         'active',
-        DefaultAccount.enabled,
+        DefaultAccount.enabled ? 'Yes' : 'No',
```

Comment:
@boubkerbribri Already create a function to don't do ternary here

Reply:
The solution could be "fillCustomersStatus` and `fillCustomersCivility`

Reply:
will be done in another PR (this fix will be deleted from this PR)